README.md

```diff
@@ -138,6 +139,32 @@ Function arguments are:
 - **`tlsVerify`** (defaults to `true`)
 
 
+### `nix2container.pullImageByManifest`
```

Comment:
I would have a preference for `pullImageFromImage` but not a strong one.

I also prefer to keep 2 separated functions for the reason you mention and also to avoid having a complex documentation and signature on the `pullImage` function.

To write `pullImage` with `pullImageFromManifest`, we would have to rely on IFD and I don't think we want this ;)

Reply:
Ah, you're right. I was thinking it would be possible without, but of course you have to know upfront the number of layers to put each in its own store path, so there's no question that IFD would be required.

> `pullImageFromImage`

Did you mean `pullImageFromManifest`, or was this intended?

Reply:
Sorry, i meant `pullImageFromManifest`.

(But if we want to loose all users, we could name it `nix2container.container2nix` :smile:)

Reply:
Renamed.